add some english notes

jiangziyan-693 opened the pull request:
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->
### PR types
<!-- One of [ New features | Bug fixes | Function optimization | Performance optimization | Breaking changes | Others ] -->

### PR changes
<!-- One of [ OPs | APIs | Docs | Others ] -->

### Description
<!-- Describe what you’ve done -->

Commit:
add some english notes

Commit:
Merge branch 'develop' of https://github.com/PaddlePaddle/Paddle into my-cool-stuff

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/Paddle?pullRequest=59198) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/Paddle?pullRequest=59198) before we can accept your contribution.<br/><hr/>**jiangziyan-693** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/Paddle?pullRequest=59198) it.</sub>

sunzhongkai588 commented on python/paddle/tensor/manipulation.py:
```diff
@@ -4861,9 +4861,10 @@ def as_real(x, name=None):
-
-    Returns a new tensor which repeats the ``x`` tensor along dimension ``axis`` using
-    the entries in ``repeats`` which is a int or a Tensor.
+    Return to a new tensor,which repeats and bulid the ``x``tensor along dimension ``axis``.
+    If the ``repeats`` is ``1-D`` Tensor,the length of the ``repeats`` must be consistent with the dimension of specified ``axis``.
+    The value of the position of ``repeats`` means the repeat times of the position of ``x``.
+    If the ``repeats`` is int,``x`` repeats ``repeats`` times along the specified ``axis``.
```
直接在原文后进行添加，即
```
    Returns a new tensor which repeats the ``x`` tensor along dimension ``axis`` using
    the entries in ``repeats`` which is a int or a Tensor.
```
后添加 If the ``repeats`` is ``1-D Tensor`` ... If the ``repeats`` is ``int`` ...

sunzhongkai588 commented on python/paddle/tensor/manipulation.py:
```diff
@@ -4861,9 +4861,10 @@ def as_real(x, name=None):
+    The value of the position of ``repeats`` means the repeat times of the position of ``x``.
+    If the ``repeats`` is int,``x`` repeats ``repeats`` times along the specified ``axis``.
 
     Args:
```
repeats参数，后面是`(Tensor or int)`，改成正规写法 `(Tensor|int)`

sunzhongkai588 reviewed (commented):
cla签署一下

jiangziyan-693 committed:
Update manipulation.py

add some english notes in "def repeat_interleave"
change repeats(Tensor or int) to (Tensor|int)

sunzhongkai588 commented on python/paddle/tensor/manipulation.py:
```diff
@@ -4861,13 +4861,15 @@ def as_real(x, name=None):
     the entries in ``repeats`` which is a int or a Tensor.
+    If the ``repeats`` is ``1-D`` Tensor,the length of the ``repeats`` must be consistent with the dimension of specified ``axis``.
+    The value of the position of ``repeats`` means the repeat times of the position of ``x``.
+    If the ``repeats`` is int,``x`` repeats ``repeats`` times along the specified ``axis``.
```
```suggestion
    If the ``repeats`` is int, ``x`` repeats ``repeats`` times along the specified ``axis``.
```

sunzhongkai588 reviewed (commented):
`` 前面得加空格，小问题～
